model/metrics/hardware/common.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  # COMMON METRICS
+  - id: metric.hardware.attributes
+    type: attribute_group
+    brief: "Attributes for hardware metrics"
+    extends: hardware.attributes.common
+    attributes:
+      - ref: hw.type
+        requirement_level: required
+
+  - id: metric.hardware.energy
+    type: metric
+    metric_name: hw.energy
+    stability: experimental
+    brief: "Energy consumed by the component, in joules"
+    instrument: counter
+    unit: "J"
+    extends: metric.hardware.attributes
+
+  - id: metric.hardware.errors
+    type: metric
+    metric_name: hw.errors
+    stability: experimental
+    brief: "Number of errors encountered by the component"
+    instrument: counter
+    unit: "{errors}"
+    extends: metric.hardware.attributes
+    attributes:
+      - ref: hw.error.type
+        requirement_level: recommended
+
+  - id: metric.hardware.power
+    type: metric
+    metric_name: hw.power
+    stability: experimental
+    brief: "Instantaneous power consumed by the component, in Watts"
```

Comment:
is it derivable from `hw.energy` ? why is this one necessary?

Reply:
You're right! It has been decided to specify both `hw.power` and `hw.energy` because:

* `hw.energy` (counter in Joules) is preferred because it provides more accurate aggregation (just like network traffic or CPU time utilization is better calculated with cumulative counters instead of averaging rates)
* Unfortunately, most systems are only capable of reporting their instantaneous electrical power consumption in Watts. To make sure existing systems all use the same metric name for electrical power, we specified `hw.power` in the semantic conventions.

Again: It can be discussed in a separate issue.